org.eclipse.xtext.ui.tests/src-gen/org/eclipse/xtext/ui/tests/editor/contentassist/services/Bug332217TestLanguageGrammarAccess.java

```diff
@@ -519,15 +519,13 @@
 		private final Keyword cRightCurlyBracketKeyword_6 = (Keyword)cGroup.eContents().get(6);
 		
 		//TableView:
-		//	'tableview' name=ID ('(' content=Parameter ')')? '{' ( ('title:' title=ScalarExpression)?
-		//		& ('titleImage:' titleImage=ImageExpression)?
-		//		& ('style:' style=TableViewStyle)? ) sections+=Section*
+		//	'tableview' name=ID ('(' content=Parameter ')')? '{' (('title:' title=ScalarExpression)? & ('titleImage:'
+		//	titleImage=ImageExpression)? & ('style:' style=TableViewStyle)?) sections+=Section*
```

Comment:
This looks worse than before @ArneDeutsch

Reply:
It looks "worse" because it is formatter ... missed the & completly before ... now it is formatted ...

Reply:
I could add preserving of newlines before and after the &. Guess that should work. Should I?

Reply:
> I could add preserving of newlines before and after the &. Guess that should work. Should I?

Yes, please. UnorderedGroups and Alternatives should behave similar in that regard and also mimic what we are doing for groups with the new-line preservation.